safekeeper/src/http/routes.rs

```diff
@@ -122,6 +124,20 @@ async fn timeline_create_handler(mut request: Request<Body>) -> Result<Response<
     json_response(StatusCode::CREATED, ())
 }
 
+/// Used only in tests to hand craft required data.
+async fn timeline_info_handler(mut request: Request<Body>) -> Result<Response<Body>, ApiError> {
```

Comment:
`timeline_info` is a bit confusing name, my first thought was that it's the same as `timeline_status`. Maybe it's better to add something about record_safekeeper_info/test_only in the name.